public/src/js/utils/observable-numeric.js

```diff
@@ -0,0 +1,16 @@
+import ko from 'knockout';
+
+export default function(initialValue) {
+    var actual = ko.observable(initialValue);
+    return ko.dependentObservable({
+        read: function ()
+        {
+            return actual();
+        },
+        write: function (newValue)
+        {
+            var parsedValue = parseFloat(newValue);
+            actual(isNaN(parsedValue) ? newValue : parsedValue);
```

Comment:
so if we can't turn it into a number, we pass along the string and allow the backend to break?